### ISSUE TITLE
CyclomaticComplexity still logs on IGNORE severity

### ISSUE THREAD

**Issue by ruby0x1** [adireddy/haxe-checkstyle#185]
The following message is printed (in purple/info level?):
`Ignore: Function "apply" is too complex (score: 21).`

After a number of "ignore" warnings the resulting output is :
`No issues found.`

The rules specify:

``` json
        {
            "type": "CyclomaticComplexity",
            "props": {
                "thresholds": [
                    {
                        "severity": "IGNORE",
                        "complexity": 20
                    },
                    {
                        "severity": "IGNORE",
                        "complexity": 30
                    }
                ]
            }
        },
```

From what I understand it - `IGNORE` is not supposed to log at all - and only logs in the case of `CyclomaticComplexity` (no other tests on IGNORE log info). My guess is a simple logging related bug that doesn't respect the IGNORE severity and just ignore it entirely. Unless there is a root severity I haven't configured that handles this?

**Comment by ruby0x1** [comment]
(I can get around the logging with `"complexity": 99` but obviously this isn't a solution :))
